org.civicrm.bootstrapcivicrm/scss/components/_page.scss

```diff
@@ -3,23 +3,45 @@
     @media (min-width: $screen-md-min) {
         overflow: hidden;
 
-        // Override of the grid widths
-        > [class*="col-"] {
+        > .row {
 
-            &:first-child {
-                width: 12.775%;
-            }
+            > [class*="col-"] {
 
-            &:last-child {
-                width: 87.2%;
+                &:first-child {
+                    max-width: $crm-page-sidebar-max-width + $grid-gutter-width;
+                }
             }
         }
 
+        // On medium/large viewports the main block must
+        // be centered and should not stretch after a certain point
+        .crm_page__main {
+            margin: 0 auto;
+            max-width: $crm-page-main-max-width;
```

Comment:
I think that media query is a bit incorrect :smile:

Even if applied, it would mean that those rules would be applied to a viewport only `180px` wide, which is obviously not what we want to do here.

Instead what we want to do is to make the first column wide max `180px` but only for medium to large viewports, which is what `@media (min-width: $screen-md-min) {` accomplishes